src/App.tsx

```diff
@@ -1,48 +1,30 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import React from 'react';
+import { Header } from './components/Header';
+import { Home } from './components/Home/Home';
+import { NotFoundPage } from './components/NotFoundPage';
+import { TabsPage } from './components/TabsPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-const App = () => (
+export const App: React.FC = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
-        </div>
-      </div>
-    </nav>
-
+    <Header />
     <div className="section">
-      <h1 className="title">Home page</h1>
-      <h1 className="title">Tabs page</h1>
-      <h1 className="title">Page not found</h1>
-
-      <div className="tabs is-boxed">
-        <ul>
-          <li data-cy="tab" className="is-active">
-            <a href="#/">Tab 1</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 2</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="tab-content">
-        Please select a tab
-      </div>
+      <Routes>
+        <Route path="/tabs-page" element={<TabsPage tabs={tabs} />}>
+          <Route path=":tabId" element={<TabsPage tabs={tabs} />} />
+        </Route>
```

Comment:
you could do the following way
```suggestion
        <Route path="/tabs-page">
        	<Route index element={<TabsPage tabs={tabs} />}> {// add 'index' property}
          <Route path=":tabId" element={<TabsPage tabs={tabs} />} />
        </Route>
```
it would be more consistent (as of link I left in the previous review)

Reply:
I got it, thanks!